hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAManagerImpl.java

```diff
@@ -67,6 +71,7 @@ public class SCMHAManagerImpl implements SCMHAManager {
 
   // this should ideally be started only in a ratis leader
   private final InterSCMGrpcProtocolService grpcServer;
+  private BackgroundSCMService trxBufferMonitorService = null;
```

Comment:
@szetszwo 
SCMHAManagerImpl is initialized before SCMContext is created in scm object. So BackgroundSCMService will have null scmContext.
To avoid this, its initialized during start of the service. Due to this reason, it can not be final and need create during start() of the SCMHAManager